applications/redis-app/src/main/java/com/bloxbean/cardano/yaci/store/redis/app/YaciStoreApplication.java

```diff
@@ -0,0 +1,32 @@
+package com.bloxbean.cardano.yaci.store.redis.app;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.info.License;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+
+@Slf4j
+@SpringBootApplication
+@ComponentScan(basePackages = "com.bloxbean.cardano.yaci.store")
+@EntityScan(basePackages = "com.bloxbean.cardano.yaci.store")
+public class YaciStoreApplication {
```

Comment:
Should we change this class name to to avoid name duplication, and make it easier to distinguish between different applications?